### ISSUE TITLE
Issues deploying openwhisk on AWS using aws-vagrant

### ISSUE THREAD

**Issue by pjdurai** [apache/openwhisk#1615]
Greetings

Has anyone deployed on AWS using aws-vagrant?
I have adopted the Vagrantfile included with openwhisk to work with AWS.
It is able to create the instance successfully and starts provisioning.
But hangs after 

=> default: Hit http://us-east-1.ec2.archive.ubuntu.com trusty/restricted Translation-en
 INFO interface: info: Hit http://us-east-1.ec2.archive.ubuntu.com trusty/universe Translation-en
 INFO interface: info: ==> default: Hit http://us-east-1.ec2.archive.ubuntu.com trusty/universe Translation-en
==> default: Hit http://us-east-1.ec2.archive.ubuntu.com trusty/universe Translation-en
 INFO interface: info: Get:40 http://ppa.launchpad.net trusty/main Translation-en [2655 B]
 INFO interface: info: ==> default: Get:40 http://ppa.launchpad.net trusty/main Translation-en [2655 B]
==> default: Get:40 http://ppa.launchpad.net trusty/main Translation-en [2655 B]

## Environment details:

* aws-vagrant,  ubuntu trusty,    
* vagrant 1.8.6

## Steps to reproduce the issue:

1. Modify the vagrantfile with AWS details. Follow instructions for aws-vagrant.   
2.  vagrant up

Has anyone done this successfully and could share some tips?

The Native Ubuntu openwhisk instructions are not working for me. So I am trying to adapt the Vagrantfile to do it on AWS.

Thanks
pj

**Comment by rabbah** [comment]
@csantanapr any input on this?

**Comment by csantanapr** [comment]
I submitted a PR to fix ubuntu native and vagrant when installing ansible. But other that should be OK. 

I'm not familiar with the vagrant on AWS only have use vagrant on Mac. 

I will suggest to do a single ubuntu VM and do a native install similar as our Travis setup. 

Those errors above look like networking problems reaching those urls, but not sure.

**Comment by rabbah** [comment]
This pr? https://github.com/openwhisk/openwhisk/pull/1644

**Comment by csantanapr** [comment]
Yes

**Comment by pjdurai** [comment]
Here is what I have found/done so far:

The Ubuntu 1404 AMI provided by Amazon is quite different from the Vagrant Box used by OpenWhisk.
I tried 'Native Ubuntu' installation on AWS for quite a while but I just couldn't get the installation to finish. It was failing in the last couple of ansible operations.

I found a (brute force) solution of creating an AWS  AMI from the exact Vagrant box used by OpenWhisk and using that as a starting point. I listed out all the Shell commands run by the Vagrant provisioning and ran them manually on this instance. And it works!

So it is a matter of pre-requisites.
